src/AzureClient/Magic/ExecuteMagic.cs

```diff
@@ -93,9 +95,9 @@ public ExecuteMagic(IAzureClient azureClient, ILogger<ExecuteMagic> logger)
                         @"
                             Run a Q# operation defined as `operation MyOperation(a : Int, b : Int) : Result`
                             on the active target in the current Azure Quantum workspace,
-                            specifying a custom job name, number of shots, timeout, and polling interval:
+                            specifying a custom job name, number of shots, timeout, polling interval, and provider-specific job parameters:
                             ```
-                            In []: %azure.submit MyOperation a=5 b=10 jobName=""My job"" shots=100 timeout=60 poll=10
+                            In []: %azure.submit MyOperation a=5 b=10 jobName=""My job"" shots=100 timeout=60 poll=10 jobParams={""Key1"":""Val1"",""Key2"":""Val2""}
```

Comment:
Should this be `--a=5 --b=10`

Reply:
The way inputs are currently parsed, adding `--` to the front would prevent the parameters from being recognized. It's a change we could theoretically make to the input parsing, but it'd be out of scope to this PR!

Reply:
Ah, that's a good point; I hadn't noticed that the way its written, the new parameter strictly prevents adding `jobParams` as a Q# parameter name. That's probably minor, but we maybe should handle that better going forward. Since doing so would be a nontrivial breaking change, let's make sure to create an issue now and defer to that issue.

Reply:
Resolving this conversation: Out of scope. The issue will be deferred.